internal/providers/github/common.go

```diff
@@ -167,6 +167,11 @@ func (c *GitHub) ListPackagesByRepository(
 	// create a slice to hold the containers
 	var allContainers []*github.Package
 
+	if c.packageListingClient == nil {
+		zerolog.Ctx(ctx).Error().Msg("No client available for listing packages")
+		return allContainers, nil
```

Comment:
Should this return an error instead?

Reply:
good point, I changed the code to return an error code from the GH function, but then catch and ignore the error with a log message on the only place where it's used.